core/code/boot.js

```diff
@@ -205,6 +205,28 @@ function prepPluginsToLoad () {
   };
 }
 
+window.sortToolbox = function () {
+  var toolboxElement = document.getElementById('toolbox') ;
+  var children = Array.prototype.slice.call(toolboxElement.children);
+  var sortedChildren = children.slice().sort(function (x, y) {
+    return x.innerText.localeCompare(y.innerText);
+  });
+  if (
+    sortedChildren.some(function (item, index) {
+      return item !== children[index];
+    })
+  ) {
+    sortedChildren.forEach(function (child) {
+      toolboxElement.removeChild(child);
+      toolboxElement.appendChild(child);
+    });
+  }
+}
+function setupToolboxSort() {
+  window.setTimeout(window.sortToolbox, 1000);
```

Comment:
The timeout won't be enough to have all the links always sorted - there'll always be possibility for some plugins to load later and add unsorted links.

Imho we should keep the observer and allow plugins to override the sorting function - that way we ensure it's always sorted (100%) and we also allow extensions by plugins 🤷

Reply:
If a plugin adds to the toolbox dynamically, it should also call sortToolbxo() after that.
If we find slow plugins that have not finished initializing, we should look into this again.